kur_kards_frontend/.eslintrc.json

```diff
@@ -1,3 +1,13 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": ["next/core-web-vitals", "airbnb", "airbnb/hooks", "prettier"],
+  "rules": {
+    "import/extensions": ["error", "never"],
+    "react/jsx-filename-extension": [
+      1,
+      { "extensions": [".js", ".jsx", ".ts", ".tsx"] }
+    ],
+    "react/jsx-props-no-spreading": "off",
```

Comment:
YESSSSSS, I don't want to guess what's in the props anymore.

Reply:
i would like to point on that this rule is for turning off the rule so that we can do it

Reply:
https://github.com/gsavchenko/kur-kards/blob/558f565d2dffe86edcc1cafa0febe19d4fbef4fe/kur_kards_frontend/pages/_app.tsx#L8